src/components/main-dashboard/weather-details/Humidity.jsx

```diff
@@ -1,9 +1,46 @@
 import React from 'react';
+import ProgressBar from '@ramonak/react-progress-bar';
+import { useWeather } from '../../WeatherProvider';
+import humidityIcon from '../../../images/humidityIcon.png';
```

Comment:
Can you elaborate on visual aesthetic and what you mean when you say visually on-theme. I know exactly how you feel though because sometimes when it comes to different designs, some approaches "work" while others don't and you really don't have an explanation for it other than it feels right and/or wrong. In fact, the humidity icon that I used was from a different website compared to the sunrise/sunset one so I can see why it looks off.

Reply:
Most websites require you to pay for SVGs but luckily, I found a website/repo that provides free animated and static weather icons that we can use. This is where I found the sun one, they also have a humidity one too if you like that better! https://basmilius.github.io/weather-icons/index-fill.html

Reply:
No I definitely agree with you. The png I picked has some dark contrasts and overall seems harsher than the ones from the repo. I fully agree with you haha! I will change it!